node/nodeTesting.go

```diff
@@ -341,7 +341,10 @@ func (n *Node) generateAndSignTxBuffArray(
 //GenerateTransaction generates a new transaction with sender, receiver, amount and code
 func (n *Node) GenerateTransaction(senderHex string, receiverHex string, value *big.Int, transactionData string) (*transaction.Transaction, error) {
 	if n.addrConverter == nil || n.accounts == nil {
```

Comment:
Please remove n.accounts from here. Might add n.addrConverter.IsInterfaceNil() check also. Here and on accounts.

Reply:
done